MAUI/Linear-Gauge/range.md

```diff
@@ -41,7 +41,7 @@ SfLinearGauge gauge = new SfLinearGauge();
 
 ## Customize range shape 
 
-A Linear Gauge range has two values to draw a range - `StartValue`, and `EndValue`.These values indicate where the range falls in the axis. In addition to this values, the shape of the range can be customized by `StartWidth`, `MidWidth` and `EndWidth` properties. To draw a line or rectangle, just `StartValue`, `StartWidth`, `EndValue` and `EndWidth` are enough - as like the above code snippet in `Default Linear Gauge Range` topic. But to draw a concave and convex like shapes, `MidWidth` property is needed. The below code snippet demonstrates how to bring a convex shape for a range.
+A Linear Gauge range has two values to draw a range - [`StartValue`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Gauges.LinearRange.html#Syncfusion_Maui_Gauges_LinearRange_StartValue), and [`EndValue`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Gauges.LinearRange.html#Syncfusion_Maui_Gauges_LinearRange_EndValue).These values indicate where the range falls in the axis. In addition to this values, the shape of the range can be customized by [`StartWidth`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Gauges.LinearRange.html#Syncfusion_Maui_Gauges_LinearRange_StartWidth), [`MidWidth`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Gauges.LinearRange.html#Syncfusion_Maui_Gauges_LinearRange_MidWidth) and [`EndWidth`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Gauges.LinearRange.html#Syncfusion_Maui_Gauges_LinearRange_EndWidth) properties. To draw a line or rectangle, just [`StartValue`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Gauges.LinearRange.html#Syncfusion_Maui_Gauges_LinearRange_StartValue), [`StartWidth`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Gauges.LinearRange.html#Syncfusion_Maui_Gauges_LinearRange_StartWidth), [`EndValue`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Gauges.LinearRange.html#Syncfusion_Maui_Gauges_LinearRange_EndValue) and [`EndWidth`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Gauges.LinearRange.html#Syncfusion_Maui_Gauges_LinearRange_EndWidth) are enough - as like the above code snippet in [`Default Linear Gauge Range`]() topic. But to draw a concave and convex like shapes, [`MidWidth`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Gauges.LinearRange.html#Syncfusion_Maui_Gauges_LinearRange_MidWidth) property is needed. The below code snippet demonstrates how to bring a convex shape for a range.
```

Comment:
Link is missing "[`Default Linear Gauge Range`]()"

Reply:
Since we can not find the exact topic, leave as it was before sir